### ISSUE TITLE
NoMethodError: undefined method `tagged_classes' for Psych:Module on Ruby 2.2.0-preview1

### ISSUE THREAD

**Issue by sferik** [dtao/safe_yaml#67]
```
irb(main):001:0> RUBY_VERSION
=> "2.2.0"
irb(main):002:0> require 'safe_yaml'
NoMethodError: undefined method `tagged_classes' for Psych:Module
    from /Users/sferik/.rbenv/versions/2.2.0-preview1/lib/ruby/gems/2.2.0/gems/safe_yaml-1.0.3/lib/safe_yaml/load.rb:43:in `<module:SafeYAML>'
    from /Users/sferik/.rbenv/versions/2.2.0-preview1/lib/ruby/gems/2.2.0/gems/safe_yaml-1.0.3/lib/safe_yaml/load.rb:26:in `<top (required)>'
    from /Users/sferik/.rbenv/versions/2.2.0-preview1/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
    from /Users/sferik/.rbenv/versions/2.2.0-preview1/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
    from /Users/sferik/.rbenv/versions/2.2.0-preview1/lib/ruby/gems/2.2.0/gems/safe_yaml-1.0.3/lib/safe_yaml.rb:1:in `<top (required)>'
    from /Users/sferik/.rbenv/versions/2.2.0-preview1/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `require'
    from /Users/sferik/.rbenv/versions/2.2.0-preview1/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `rescue in require'
    from /Users/sferik/.rbenv/versions/2.2.0-preview1/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:39:in `require'
    from (irb):2
    from /Users/sferik/.rbenv/versions/2.2.0-preview1/bin/irb:11:in `<main>'
```

I do not get this error on Ruby 2.1.2, 2.0.0, or 1.9.3.

**Comment by pdostal** [comment]
I had this error after I anstalled jekyll and after I installed `safe_yaml` everything's OK, so thanks :+1:

**Comment by willgdjones** [comment]
Getting this error again with Ruby 2.2.3 and safe_yaml 1.0.3 and running jekyll 2.5.3

**Comment by noroot** [comment]
confirm, bug still appears, ```~/.rvm/gems/ruby-2.2.3/gems/safe_yaml-1.0.3/lib/safe_yaml/load.rb:43:in `<module:SafeYAML>': undefined method `tagged_classes' for Psych:Module (NoMethodError)```

**Comment by noroot** [comment]
Fixed with 1.0.4 `safe_yaml` version
